### DISCUSSION TITLE
Cd-ing into the directory of a file path

### DISCUSSION THREAD

**Discussion by halloleo** [ajeetdsouza/zoxide#836] [Q&A] [answered]
Somtikes I have the full path to a file like

    /Users/me/project/data/extra/info.csv

and I hoped that

    $ z /Users/me/project/data/extra/info.csv

cd-ed into `/Users/me/project/data/extra/`. However I get always "no match found".

Can `zoxide` do this? Have I just missed how? 

If zoxide can't do this (yet), is this feature in scope for zoxide?

**Comment by ajeetdsouza** [comment, marked as answer]
This is not a planned feature as of now - you cannot `cd` into a file.
